math/mathcore/inc/TMath.h

```diff
@@ -508,6 +511,19 @@ struct Limits {
    Double_t VavilovI(Double_t x, Double_t kappa, Double_t beta2);
    Double_t Voigt(Double_t x, Double_t sigma, Double_t lg, Int_t r = 4);
 
+   
+   
+   //Vectorized Functions
+   ::ROOT::Double_v Log2(::ROOT::Double_v &x);
+   ::ROOT::Double_v BreitWigner(::ROOT::Double_v &x, Double_t mean=0, Double_t gamma=1);
+   ::ROOT::Double_v Gaus(::ROOT::Double_v &x, Double_t mean=0, Double_t sigma=1, Bool_t norm=kFALSE);
+   ::ROOT::Double_v LaplaceDist(::ROOT::Double_v &x, Double_t alpha=0, Double_t beta=1);
+   ::ROOT::Double_v LaplaceDistI(::ROOT::Double_v &x, Double_t alpha=0, Double_t beta=1);
+
+   ::ROOT::Double_v Freq(::ROOT::Double_v &x);
+   ::ROOT::Double_v KolmogorovProb(::ROOT::Double_v &z);
```

Comment:
In [this](https://github.com/root-project/root/pull/2236#issuecomment-400722627) comment, I explained that when Vc is disabled, `ROOT::Double_v` is actually just `double`, so in that case you are redefining the functions here, which breaks compilation.

Reply:
Yes, the #ifdef R__HAS_VECCORE needs to be present here so that it doesn't compile if that is not defined.
As per PR heading, it should not be tested just yet.